package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@threshold-network/solidity-contracts",
-  "version": "1.1.0-dev",
+  "version": "1.1.0-alpha",
```

Comment:
I think we should bump it to `1.2.0-dev` because all automated deployment flows we have expects `-dev` suffix AFAIK. Also, I'd bump it separately from this PR so that we can tag `v1.1.0` with the right version of deployment scripts. So what I'd do:

- review/merge this PR without bumping up version in `package.json`,
- create `releases/mainnet/v1.1.0` release branch,
- update dependencies to the ones from mainnet and version in `package.json` to `1.1.0` there and tag it as `v1.1.0`,
- create a PR that will bump up version to `1.2.0-dev` here.

Reply:
Can you clarify what do you mean by `update dependencies to the ones from mainnet`? As for the rest, I think it's a good plan. We can leave the published 1.1.0-alpha just as a way to unblock the dashboard team.

Reply:
We have a dependency to `keep-core` that is pointing to `-dev` version on `main` (and I see it's also pointing to `-dev` in alpha package). We need to update dependency on the release branch to make sure correct contracts are resolved during the deployment and to have the mainnet NPM package referring to mainnet keep-core NPM package. This is for v1.0.0: https://github.com/threshold-network/solidity-contracts/blob/v1.0.0/package.json#L59-L66